website/src/components/SearchPage/DownloadDialog/DownloadDialog.tsx

```diff
@@ -21,7 +22,7 @@ type DownloadDialogProps = {
     referenceGenomesLightweightSchema: ReferenceGenomesLightweightSchema;
     allowSubmissionOfConsensusSequences: boolean;
     dataUseTermsEnabled: boolean;
-    metadata: Metadata[];
+    schema: Schema;
```

Comment:
I don't think this change is required? we only use the schema.metadata item here if I am not mistaken?

Reply:
ah sorry it is required for the FieldSelectorModal